packages/recomponents/src/components/r-date-input/r-calendar-manager.vue

```diff
@@ -256,6 +257,7 @@
                     return;
                 }
                 this.$emit('input', date);
+                this.$refs.datePicker.hidePopover();
```

Comment:
Isn't it better to have an option for closing on select with the default value being true?